content/wasm-functions/get-logs.md

```diff
@@ -8,20 +8,18 @@ enable_shortcodes = true
 - [Retrieving Logs of Your Application](#retrieving-logs-of-your-application)
 - [Next Steps](#next-steps)
 
-Retrieving logs of your Spin applications at runtime is crucial to understand runtime behavior and for gathering insights. 
+Retrieving logs of your Spin applications at runtime is crucial to understand runtime behavior and for gathering insights.
 
 ## Retrieving Logs of Your Application
 
 Use the `spin aka logs` command, to retrieve logs of your Spin application deployed to _Fermyon Wasm Functions_.
 
-If you invoke the command from within the directory of your Spin application (the directory in which your `spin.toml` file is located), it will - by default - load all logs of the current Spin application.
-
-If the command is invoked outside the application directory, or if you want to receive log messages of a particular Spin application running on _Fermyon Wasm Functions_, you must explicitly specify the name of the application using the `--app` flag:
+By default it will provide logs for the application that your workspace is [linked to](app-linking). If your workspace is not linked to an application, you can specify the application name using the `--app-name` flag.
```

Comment:
Clarified why a user would want to pass just the `--app-name` flag vs link their workspace
```suggestion
By default, this command provides logs for the application your workspace is [linked to](app-linking). If your workspace isn't linked to an application, you can either specify the application name using the `--app-name` flag for a one-time connection or run `spin aka link` to link your workspace to an application running on _Fermyon Wasm Functions_.
```